fcrepo-persistence-ocfl/src/main/java/org/fcrepo/persistence/ocfl/impl/OCFLPersistentStorageUtils.java

```diff
@@ -338,4 +352,21 @@ public static String mintOCFLObjectId(final String fedoraIdentifier) {
 
         return bareFedoraIdentifier;
     }
+
+    /**
+     * Create a new ocfl repository
+     * @param ocflStorageRootDir The ocfl storage root directory
+     * @param ocflWorkDir The ocfl work directory
+     * @return the repository
+     */
+    public static MutableOcflRepository createRepository(final File ocflStorageRootDir, final File ocflWorkDir) {
+        ocflStorageRootDir.mkdirs();
+        ocflWorkDir.mkdirs();
```

Comment:
Please add log message indicating location of `storageRootDir` and `workDir`.
..like: https://github.com/fcrepo4/fcrepo4/pull/1625/files#diff-f2c25ea789161a21300d5fa03b8a3965L71-L72